src/app/wallet/wallet.component.html

```diff
@@ -168,7 +155,7 @@ <h4 class="init-left__title">
                                                 <span class="info__state-failed" *ngIf="transaction.state === 'FAILED'">
                                                     Failed
                                                 </span>
-                                                {{ transaction.date | date:'medium' }}
+                                                {{ transaction.date | date: "medium" }}
```

Comment:
Revert this back. Use convention as Angular community uses it (look [here](https://angular.io/guide/pipes#transforming-data-with-parameters-and-chained-pipes)).